jwst/lib/tests/build_engdb.ipynb

```diff
@@ -66,6 +67,72 @@
     "# Main"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Produce a cache for proposal jw01029\n",
+    "\n",
+    "This is using the `EngdbMast` caching method.\n",
+    "Target exposure is jw01029008001_gs-acq2_2022143003622"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "db_path = Path('./data/engdb_jw01029')\n",
+    "try:\n",
+    "    shutil.rmtree(db_path)\n",
+    "except FileNotFoundError:\n",
+    "    pass\n",
+    "db_path.mkdir()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# date_start = Time('2022-05-23T00:36:08.000')\n",
+    "# date_end = Time('2022-05-23T00:36:22.480')\n",
+    "# date_start = Time('2022-05-22T18:05:00')\n",
+    "# date_end = Time('2022-05-22T18:16:00')\n",
+    "date_start = Time('2022-05-22T18:05:54.')\n",
+    "date_end = Time('2022-05-22T23:48:39')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#date_start = Time('2022-05-23T00:35:0')\n",
+    "#date_end = Time('2022-05-23T00:37:0')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "engdb = EngdbMast(token='8fb3131048d448f9997f5d76df74e9b3')"
```

Comment:
Should this MAST token be included here in a public notebook?

Reply:
Good catch. Oh probably not, though it is also stale. Will remove. 😱